carmin-server/server/test/resources/test_executions.py

```diff
@@ -0,0 +1,101 @@
+import pytest
+import os
+import json
+from server import app
+from server.common.error_codes_and_messages import (
+    EXECUTION_IDENTIFIER_MUST_NOT_BE_SET, INVALID_PIPELINE_IDENTIFIER,
+    INVALID_MODEL_PROVIDED)
+from server.resources.models.pipeline import PipelineSchema
+from server.resources.models.execution import ExecutionSchema
+from server.test.fakedata.pipelines import PIPELINE_FOUR
+from server.test.fakedata.executions import (
+    POST_VALID_EXECUTION, POST_INVALID_EXECUTION_FILE_NOT_EXIST,
+    POST_INVALID_EXECUTION_ARRAY_FILE_NOT_EXIST, POST_INVALID_IDENTIFIER_SET,
+    POST_INVALID_EXECUTION_IDENTIFIER_NOT_EXIST, POST_INVALID_MODEL)
+from server.test.fakedata.users import standard_user
+from server.test.utils import get_test_config, error_from_response
+
+
+@pytest.yield_fixture()
+def test_config(tmpdir_factory):
+    test_config = get_test_config()
+    test_config.db.session.add(standard_user())
```

Comment:
As you use directly the ApiKey, this does not cause issues, but please use the encrypted=True (or simply True) to have this test user password encrypted in the DB. @louis-ver Otherwise, I could maybe try to do this encryption as a trigger on INSERT to have the password changed seemlessly, but as we only use the password in two endpoints like this (authenticate and register), this is still correct as it is. However, please add the True parameter.